### PR TITLE
Dawn/adds checklist

### PR DESCRIPTION
**Pull request by dawnkelly09**
This PR adds a document to the style guide repo with a checklist for writers. This checklist is intended to help writers screen for commonly missed items before opening a PR for review.

### CODE REVIEW

File: pr-checklist.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Documentation checklist
+
+Before requesting a PR review, please check your work against this list of commonly missed items from the style guide. 
+
+Guidance for Grammarly, Screenshots, and Diagrams can be found in the [Docs Standards](https://papermoon.atlassian.net/wiki/spaces/DR/pages/885261/Docs+Standards){target=\_blank}
```
Comment (albertov19) [inline pr-checklist.md:5]: Here you are linking to a confluence page which is not public. Maybe we should migrate that to this Github?

File: pr-checklist.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	- [ ] Ensure lists requiring a specific order (step-by-step instructions, etc.) are numbered
+	- [ ] Use bullets for lists that can be in any order (list of features, etc.)
+	- [ ] Check your formatting for list items - they should look like:
+		- Item name - copy about the item/description
+			- [ ] Item name capitalized
+			- [ ] space, dash, space between item name and description
+			- [ ] the description starts with a lowercase letter
+			- [ ] there is no punctuation at the end of the description
+- [ ] Check your links
+	- [ ] Test all links to ensure they open to the expected content
```
Comment (albertov19) [inline pr-checklist.md:17]: I don't think this is scalable tho, in the past we've used crawlers to check for URL checking

File: pr-checklist.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	- [ ] Ensure all links are followed by `{target=\_blank}` so they will open in a new tab when selected
```
Comment (albertov19) [inline pr-checklist.md:18]: This is not mandatory tho. I think we suggest doing it if you are going to another page, but if you are heading to another section within the same page we don't use it

**Review by albertov19 — CHANGES_REQUESTED** [review]
Looks great! Left some comments.

What about description min and max length? And title min and max length? That is something I'm always missing hahaha

File: pr-checklist.md (hunk 1 above)
Comment (dawnkelly09) [inline pr-checklist.md:5]:
```suggestion
Guidance for Grammarly, Screenshots, and Diagrams can be found in the internal Docs Standards document.
```

File: pr-checklist.md (hunk 2 above)
Comment (dawnkelly09) [reply pr-checklist.md:17]: It's a big job for an entire site or section, but it shouldn't be over one ticket. I check each link as I add it to the content I'm creating so this is easy for me to check off. At some point in the content creation process, the writer has to make sure the link works.

**Review by dawnkelly09 — COMMENTED** [review]
Updated the confluence link per our conversation + a comment on link checking. Thanks!

**Comment by dawnkelly09** [comment]
Closing PR for revert of these commits

**Pull request closed without merging**